Tests/Fixtures/TestProject/Project.xcodeproj/project.pbxproj

```diff
@@ -7,845 +7,842 @@
 	objects = {
 
 /* Begin PBXAggregateTarget section */
-		AT_445731917037 /* SuperTarget */ = {
+		AB59C0690172FF119F045336C1F1CA0D /* SuperTarget */ = {
```

Comment:
Sad to see the prefix gone on these 😢.

Reply:
Yeah, maybe we can see about adding them back in in xcodeproj

Reply:
I'm open to suggestions. Is there value in having the prefix?

Reply:
When debugging in code, and we print out the string, it's been useful to see the type of the reference.